testing/src/test/java/org/oppia/android/testing/logging/EventLogSubjectTest.kt

```diff
@@ -0,0 +1,298 @@
+package org.oppia.android.testing.logging
+
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.oppia.android.app.model.AppLanguageSelection
+import org.oppia.android.app.model.AudioTranslationLanguageSelection
+import org.oppia.android.app.model.EventLog
+import org.oppia.android.app.model.EventLog.ExplorationContext
+import org.oppia.android.app.model.EventLog.TopicContext
+import org.oppia.android.app.model.OppiaLanguage
+import org.oppia.android.app.model.ProfileId
+import org.oppia.android.app.model.WrittenTranslationLanguageSelection
+
+@RunWith(JUnit4::class)
+class EventLogSubjectTest {
+  @Test
+  fun testHasTimeStamp_withTimeStamp_matchesTimeStamp() {
```

Comment:
All the tests in this class needs to be renamed to follow our preferred format: https://github.com/oppia/oppia-android/wiki/Oppia-Android-Testing#guidelines-for-testing, e.g. `testEventLogSubject_matchesCorrectTimestamp`
`testEventLogSubject_failsOnUnmatchingTimestamp`